acceptance-tests/src/test/java/me/atam/atam4jsampleapp/PassingTestAcceptanceTest.java

```diff
@@ -47,7 +58,24 @@ public void givenPassingTest_whenTestsEndpointCalledAfterTestRun_thenOKMessageRe
         );
     }
 
+    /*
+        TODO - Currently this test relies on a junit test that "knows" if it is being run or not.  Once/if we build support for exposing the if
+        a test run is in progress then this test's implementation can change.
+    */
+    @Test
+    public void givenPassingTest_whenTestsEndpointCalledDuringTestRun_thenStatusOfLastRunReturned() throws InterruptedException {
+        //given test run completed
+        dropwizardTestSupportAppConfig = Atam4jApplicationStarter.startApplicationWith(0, TestThatKnowsIfItsBeingRun.class, 1000);
+        TestsRunResult firstTestRunResult = getResponseFromTestsEndpointOnceTestsRunHasCompleted().readEntity(TestsRunResult.class);
 
+        //when tests are in progress
+        new PollingPredicate(100, 10, o -> TestThatKnowsIfItsBeingRun.testInProgress(), () -> null).pollUntilPassedOrFail("Tests were never seen to be In Progress");
```

Comment:
Compiler warning on this line.
```
 Unchecked call to 'PollingPredicate(int, int, Predicate<T>, Supplier<T>)' as a member of raw type 'me.atam.atam4j.PollingPredicate' (at line 69) 
```

Should be easily fixed by adding the diamond operator. `new PollingPredicate<>`